crates/hyperswitch_connectors/src/connectors/bamboraapac/transformers.rs

```diff
@@ -654,8 +667,8 @@ impl<F>
 }
 
 // refund body in soap format
-pub fn get_refund_body(
-    req: &BamboraapacRouterData<&types::RefundExecuteRouterData>,
+pub fn get_refund_body<F>(
+    req: &BamboraapacRouterData<&types::RefundsRouterData<F>>,
```

Comment:
Undo this change and add type `type RefundExecuteRouterData` in `crates/hyperswitch_domain_models/src/types.rs` instead